Android/BetterPickers/External-Dependency-Info.txt

```diff
@@ -0,0 +1,52 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for BetterPickers incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# android-betterpickers
+# https://github.com/code-troopers/android-betterpickers
+########################################
+
+Copyright 2013 Derek Brameyer, Code-Troopers
+
```

Comment:
You need to add the following...  

 - Copyright (C) 2013 The Android Open Source Project
 - Copyright (C) 2012 Google Inc.
 - Copyright (C) 2012 Jake Wharton
 - Copyright (C) 2011 Patrik Akerfeldt